modules/telco-core-rds-components.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
```

Comment:
I wondered if this modules was missing any information, so I looked up the [preview](https://93530--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/telco-core-rds.html#telco-core-rds-components). It seems that this module would better serve as an `ASSEMBLY` that included the modules that follow.